.github/workflows/build-and-test.yml

```diff
@@ -50,70 +65,71 @@ jobs:
       - name: Run Affected build
         run: yarn nx affected:build --exclude=docs --parallel=false
 
-      - run: yarn nx-cloud stop-all-agents
+      - run: npx nx-cloud stop-all-agents
```

Comment:
this will be problematic for the build-docs step. We could introduce another job which just stops all agents. It needs to wait for `build` and `build-docs` to pass

Reply:
Until we can upgrade to Nx >=13.10 to support Node.js v16 across all projects, we have to keep the `docs` build in a job separate from the `nx affected` job for the other projects.

> We could introduce another job which just stops all agents. It needs to wait for build and build-docs to pass

That's a good suggestion 👍

Reply:
Aah, now I understand this custom Nx Cloud DTE workflow. We don't need DTE to build just the `docs` project. We can still use Nx Cloud's distributed cache or build the `docs` project on Nx Cloud cache miss.

Reply:
I have disabled DTE for the `build-docs` job as it can't run on the Node.js 14 agents which is the reason it has a separate job. It can still use Nx Cloud's distributed cache though.